tools/mo/openvino/tools/mo/front/onnx/roialign_ext.py

```diff
@@ -17,8 +17,14 @@ def extract(cls, node):
         output_width = onnx_attr(node, 'output_width', 'i', default=1)
         sampling_ratio = onnx_attr(node, 'sampling_ratio', 'i', default=0)
         spatial_scale = onnx_attr(node, 'spatial_scale', 'f', default=1.0)
-
-        ROIAlign.update_node_stat(node, {'pooled_h': output_height, 'pooled_w': output_width,
-                                         'sampling_ratio': sampling_ratio, 'spatial_scale': spatial_scale,
-                                         'mode': mode})
+        onnx_opset_version = get_onnx_opset_version(node)
+        if onnx_opset_version >= 16:
+            aligned_mode = onnx_attr(node, 'coordinate_transformation_mode', 's', default=b'half_pixel').decode()
```

Comment:
It won't work for onnx ROIAlign-16 `coordinate_transformation_mode` set to "output_half_pixel".
We need to map it to one of the openvino ROIAlign-9 `aligned_mode` options.

Please take a look at support for ONNX ROIAlign-16 from new ONNX FE (onnx importer):
https://github.com/openvinotoolkit/openvino/blob/5fa669785c555838df8b3cdc3fe24a644c91d59c/src/frontends/onnx/frontend/src/op/roi_align.cpp#L64-L66